demo/demoScreen.js

```diff
@@ -141,6 +163,7 @@ export default class KeyboardInput extends Component {
           onItemSelected={this.onKeyboardItemSelected}
           onKeyboardResigned={this.onKeyboardResigned}
           revealKeyboardInteractive
+          useSafeArea={this.state.useSafeArea}
```

Comment:
This does not actually change the keyboard, I'm guessing this is why the other prop is called `initialProps`, maybe it cannot be changed after the view has been constructed.
There's a simple way to solve this in the screen - change the `key` of `KeyboardAccessoryView` but this will re-construct the view and so it will be closed - not very nice to look at.
Is there another way to solve it? Can the prop be changed mid run?

Reply:
I think we can pass update when useSafeArea is changed - to the native code that will only update the constraint.

Reply:
Done